UI/index.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/html">
+<head>
+    <meta charset="UTF-8">
+    <title>Sweet Tweet</title>
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/fontawesome.min.css" rel="stylesheet" />
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/solid.min.css" rel="stylesheet" />
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <meta name="viewport" content="width, initial-scale=1.0">
+</head>
+
+<body>
+<header>
+     <h1> Sweet Tweet </h1>
+</header>
+<main>
+    <div class="field-user">
+        <div>
+            <img class="avatar" src="91e83b00c27bf4d5bb849a6ac2b81fe5.jpg">
+            Alexander T
+        </div>
+        <div>
+            <span><button class = "button-add">Add post</button></span>
+            <span><button class = "button-log">Log out</button></span>
+        </div>
+    </div>
+
+    <div class="mobile-filtering">
+        <div class="filters-selected">Filters that are used</div>
+        <button class="mobile-filtering--button">
+            <i class="fas fa-filter"></i>
+        </button>
+    </div>
+<div class="main-field">
+    <div class="filtering"> <h2> Filtering posts</h2>
+        <p>
```

Comment:
There are a lot of `br` tags used, but from my POV better to achieve something similar via CSS